content/articles/working-with-styled-components-in-react/index.md

```diff
@@ -0,0 +1,504 @@
+         <Content>
+     </Container>);
+}
+export default Landing;
+```
+
+To create our second component, open the `Header.js` file in the component folder. Add the following code to the file:
+
+```JavaScript
+
+import styled from "styled-components";
+
+// This component will render a Navbar before styling.
+const Header = (props)=>{
+    return (
+    <Nav>
+        Header
+    </Nav>
+    );
+}
+
+export default Header;
+```
+
+#### Routing a Component into the main App
+
+To get things started, let's open `App.js`  which is the base of our application. We replace all the content in it with the code below to create an App function that will load our styled-components and render them as the landing page.
+
+After building components/pages in your web application, you may need to expose and let your users navigate through them. To achieve this, you need a dedicated router. [React Router](https://www.freecodecamp.org/news/react-router-cheatsheet/) is a suite of navigational packages/components such as react-router-dom. 
+
+To route components into the main App, you will import some features from the `react-router-dom`, a React-router package that we installed earlier.
+
+```JavaScript
+Import {BrowserRouter as Router, Switch, Route} from "react-router-dom";//The Router, Switch, and Route will help us move between our created component and the main App.js.
+Import Landing from "./components/Landing" //This is to import the component created in the Landing.js file.
+Import Header from "./components/Header"  // This is to import the component created in the Header.js file.
+Import './App.css';//Load a set of predefined CSS that will define how HTML elements in the landing page behave.
+ 
+function App() {   //main app
+   return(
+  <div>
+   <Router>
+     <Switch>
+       <Route exact path="/">
+      
+       </Route>
+     </Switch>
+   </Router>
+  </div>)
+
+export default App; 
+
+```
+This component will render a `Navbar` before styling.
+
+Now that we have created our components, it is time to route it into the `App.js`. To do so, add the code below between the *Route tags* as shown below:
+
+```JavaScript
+ <Landing/>
+```
+```JavaScript
+ <Header/>
+```
+The final `App.js` should now contain the following code:
+
+```JavaScript
+
+import {BrowserRouter as Router, Switch, Route} from "react-router-dom";
+import Landing from "./components/Landing";
+import Header from "./components/Header";
+import './App.css'
+
+function App() {
+  return(
+  <div>
+   <Router>
+     <Switch>
+       <Route path="/">
+         <Landing/>
+         <Header/>
+       </Route>
+     </Switch>
+   </Router>
+  </div>)
+}
+
+export default App;
+```
+![App.js final](/engineering-education/working-with-styled-components/App.js.png)
+
+### Onto some styling now
+
+Be sure to create an images folder inside the public folder of the application.
+
+You should have the landing page background image, disney+ icon image, logo-one, and logo-two within the folder.
```

Comment:
Add links to the images that you have used. That would be better than me going to search for images on the site mentioned.